{"name": "sucessfully login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument\n  (Session info: chrome=120.0.6099.71)\nStacktrace:\n\tGetHandleVerifier [0x00007FF665B54D02+56194]\n\t(No symbol) [0x00007FF665AC04B2]\n\t(No symbol) [0x00007FF6659674C3]\n\t(No symbol) [0x00007FF665950F3B]\n\t(No symbol) [0x00007FF66594EE85]\n\t(No symbol) [0x00007FF66594F603]\n\t(No symbol) [0x00007FF66596A0FD]\n\t(No symbol) [0x00007FF6659F2C91]\n\t(No symbol) [0x00007FF6659D5E7A]\n\t(No symbol) [0x00007FF6659F2786]\n\t(No symbol) [0x00007FF6659D5C23]\n\t(No symbol) [0x00007FF6659A4A45]\n\t(No symbol) [0x00007FF6659A5AD4]\n\tGetHandleVerifier [0x00007FF665ECD5BB+3695675]\n\tGetHandleVerifier [0x00007FF665F26197+4059159]\n\tGetHandleVerifier [0x00007FF665F1DF63+4025827]\n\tGetHandleVerifier [0x00007FF665BEF029+687785]\n\t(No symbol) [0x00007FF665ACB508]\n\t(No symbol) [0x00007FF665AC7564]\n\t(No symbol) [0x00007FF665AC76E9]\n\t(No symbol) [0x00007FF665AB8094]\n\tBaseThreadInitThunk [0x00007FF909B7257D+29]\n\tRtlUserThreadStart [0x00007FF90B48AA58+40]\n\n", "trace": "  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_step_definitions.py\", line 11, in step_impl\n    context.driver.get(ConfigReader.read_configuration(\"basic info\", \"url\"))\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given that the user is on the login site", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument\n  (Session info: chrome=120.0.6099.71)\nStacktrace:\n\tGetHandleVerifier [0x00007FF665B54D02+56194]\n\t(No symbol) [0x00007FF665AC04B2]\n\t(No symbol) [0x00007FF6659674C3]\n\t(No symbol) [0x00007FF665950F3B]\n\t(No symbol) [0x00007FF66594EE85]\n\t(No symbol) [0x00007FF66594F603]\n\t(No symbol) [0x00007FF66596A0FD]\n\t(No symbol) [0x00007FF6659F2C91]\n\t(No symbol) [0x00007FF6659D5E7A]\n\t(No symbol) [0x00007FF6659F2786]\n\t(No symbol) [0x00007FF6659D5C23]\n\t(No symbol) [0x00007FF6659A4A45]\n\t(No symbol) [0x00007FF6659A5AD4]\n\tGetHandleVerifier [0x00007FF665ECD5BB+3695675]\n\tGetHandleVerifier [0x00007FF665F26197+4059159]\n\tGetHandleVerifier [0x00007FF665F1DF63+4025827]\n\tGetHandleVerifier [0x00007FF665BEF029+687785]\n\t(No symbol) [0x00007FF665ACB508]\n\t(No symbol) [0x00007FF665AC7564]\n\t(No symbol) [0x00007FF665AC76E9]\n\t(No symbol) [0x00007FF665AB8094]\n\tBaseThreadInitThunk [0x00007FF909B7257D+29]\n\tRtlUserThreadStart [0x00007FF90B48AA58+40]\n\n", "trace": "  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_step_definitions.py\", line 11, in step_impl\n    context.driver.get(ConfigReader.read_configuration(\"basic info\", \"url\"))\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Amph4rd\\PycharmProjects\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1702164642579, "stop": 1702164642583}, {"name": "When he types login data", "status": "skipped", "start": 1702164644646, "stop": 1702164644646}, {"name": "Then will see the dashboard page", "status": "skipped", "start": 1702164644646, "stop": 1702164644646}], "start": 1702164637605, "stop": 1702164644646, "uuid": "82af6387-6b15-4cbe-b0b0-d0d3e31cce11", "historyId": "355f47e0a17298803b3f2e8c865673f9", "fullName": "login on the site: sucessfully login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "login on the site"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}